### DISCUSSION TITLE
Is there a way to directly change camera matrix?

### DISCUSSION THREAD

**Discussion by copark86** [raysan5/raylib#2045] [Q&A] [answered]
Is there a way to directly change the camera matrix value instead of changing up, target, position? I know there is a function GetCameraMatrix but I can't find SetCameraMatrix. Should I just extract Rz(target), Ry(up), XYZ(position) from the reference matrix and set it by changing up, target, position?
How should I approach this problem?

**Comment by raysan5** [comment, marked as answer]
@copark86 Excuse for the super-late resposne, I missed this discussion. `raylib` has no mechanism to do that but `rlgl` (that is a bit lower level than raylib and can also be used in conjunction with raylib) has `rlSetMatrixModelview()` to set a custom modelview matrix at a desired moment.
